uiowa/uiowa
# bulk publishing permissions

[Admissions wants to be able to bulk un-piblish/publish items](https://github.com/uiowa/uiowa/issues/3610)


# How to test

1. Check out this branch.
2. Sync admissions.
3. Check that the bulk operations are enabled in permissions for webmaster and publisher.
4. try masquerading as webmaster or publisher and un-piblishing/publishing some content in bulk.
5. try masquerading as an editor and see that you cannot do bulk un-piblishing/publishing.
